psalm-baseline.xml

```diff
@@ -1400,32 +1813,48 @@
       <code>true</code>
       <code>true</code>
     </InvalidArgument>
+    <UnusedMethodCall>
+      <code>setAccessible</code>
+    </UnusedMethodCall>
   </file>
   <file src="test/Generator/TypeGenerator/CompositeTypeTest.php">
-    <MixedInferredReturnType occurrences="1">
-      <code>iterable</code>
-    </MixedInferredReturnType>
+    <PossiblyUnusedMethod>
+      <code>validType</code>
+    </PossiblyUnusedMethod>
+  </file>
+  <file src="test/Generator/TypeGenerator/IntersectionTypeTest.php">
+    <PossiblyUnusedMethod>
+      <code>invalidIntersectionsExamples</code>
+      <code>sortingExamples</code>
+    </PossiblyUnusedMethod>
+  </file>
+  <file src="test/Generator/TypeGenerator/UnionTypeTest.php">
+    <PossiblyUnusedMethod>
+      <code>invalidUnionsExamples</code>
```

Comment:
I think the Psalm extension does not yet understand the new attributes syntax

Reply:
Indeed, I'll report a bug in psalm then

Reply:
Thanks! For now we can proceed here regardless :)